qgis_deployment_toolbelt/utils/win32utils.py

```diff
@@ -298,6 +307,79 @@ def set_environment_variable(
         return False
 
 
+def read_registry_value(
+    key: tuple, value_name: str, access_mode: Literal["read", "write"] = "read"
+) -> str | None:
+    r"""Read a value from the Windows registry.
```

Comment:
Is the `r` prefix really required? In recent Python versions, I though special char are supported, no?